hooks/useFiles.web.ts

```diff
@@ -0,0 +1,7 @@
+export const useFiles = async (filetype: string) => {
+  // this is a walk around for es lint error
+  if(!filetype){
+    return undefined;
```

Comment:
You have a bunch of lint errors in this file, please fix

Reply:
Updated